bottlerocket/samples/Makefile.toml

```diff
@@ -9,7 +9,13 @@ CLUSTER_TYPE = { value = "eks", condition = { env_not_set = ["CLUSTER_TYPE"] } }
 ASSUME_ROLE = { value = "~", condition = { env_not_set = ["ASSUME_ROLE"] } }
 AWS_REGION = { value = "us-west-2", condition = { env_not_set = ["AWS_REGION"] } }
 UPGRADE_VERSION = { value = "v1.11.1", condition = { env_not_set = ["UPGRADE_VERSION"] } }
-STARTING_VERSION = { value = "v1.11.0", condition = { env_not_set = ["STARTING_VERSION"] } }
+STARTING_VERSION = { script = ["""\
+        aws ssm get-parameter \
+            --region us-west-2 \
+            --name "/aws/service/bottlerocket/aws-k8s-1.24/arm64/latest/image_version" \
```

Comment:
Is the `aws-k8s-1.24` what we target to use on testsys? I was thinking if we deprecate `aws-k8s-1.24` later, will we need update here?

Reply:
Yes we use `aws-k8s-1.24` for all of our eks testing.